CHANGELOG.rst

```diff
@@ -2,11 +2,17 @@
 Changes
 =======
 
-1.x.x (TBD)
------------
+1.8.0 (2024-01-06)
+------------------
+
 **Bugfixes and QOL**
 
-* [#66] updated commonground-api-common to 2.1.0 (TBD)
+* [maykinmedia/open-api-framework#66] Update zgw consumers
+* [maykinmedia/open-api-framework#46] Upgrade open-api-framework to 0.9.1
+* [#199] Upgrade mozilla-django-oidc-db to 0.21.1
+* [#200] Upgrade commonground-api-common to 2.2.0
```

Comment:
The warning below is related to this library, could you move this line to the bottom of the Bugfixes list?